WtWSMS/common/dynasties/BP_dynasties.txt

```diff
@@ -483,7 +483,7 @@
 	culture =  geatish
 }
 1042188 = {
-	name =  "dynn_Crimling"
+	name =  "dynn_Agalling"
```

Comment:
I can't find anything about the old dynasty name, but I don't see any content around the new one either?

Reply:
Both are fictional. A name with a turkish etymology was replaced by another that refers to the founding character of the family.

Reply:
Ok, good.